.github/workflows/pull_request_push_test.yml

```diff
@@ -32,6 +32,28 @@ jobs:
           distribution: "temurin"
       - name: Run tests
         run: sbt clean && sbt test
+
+  eslint:
+    name: eslint
```

Comment:
>name: eslint [](http://example.com/codeflow?start=4&length=12)

Please add trigger condition so eslint and prettier formatter only fires when PR includes change for ui files.

Reply:
I created another file to address this issue - it'll better help to separate API checks and UI checks in the workflow for readability and maintainability